spec/system/candidate_interface/candidate_expire_session_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+describe 'Candidate session expires' do
+  it 'can expire the current candidate session after 30 minutes' do
```

Comment:
in this case present tense is preferable:
`it 'expires the current candidate session after 30 minutes' do`

Reply:
I suggest also not specifying the duration in here (so we don't need to update the spec name in the future)